frontend/lib/resolution/default-functions.cpp

```diff
@@ -76,18 +76,22 @@ areOverloadsPresentInDefiningScope(Context* context, const Type* type,
 
   auto haveQt = QualifiedType(QualifiedType::VAR, type);
 
-  // loop through IDs and see if any are methods on the same type
+  // loop through IDs and see if any are methods or operators (method or
+  // standalone) on the same type
   for (auto& ids : vec) {
     for (const auto& id : ids) {
       auto node = parsing::idToAst(context, id);
       CHPL_ASSERT(node);
 
       if (auto fn = node->toFunction()) {
-        if (fn->isMethod()) {
+        if (fn->isMethod() || fn->kind() == Function::Kind::OPERATOR) {
           ResolutionResultByPostorderID r;
           auto vis = Resolver::createForInitialSignature(context, fn, r);
-          fn->thisFormal()->traverse(vis);
-          auto receiverQualType = vis.byPostorder.byAst(fn->thisFormal()).type();
+          // receiver is 'this' for method, first formal for standalone operator
```

Comment:
The first formal isn't the receiver; "receiver" as a term means the thing bound to `this`. This is just a terminology thing though and is best resolved by adjusting the comment & renaming `receiverFormal`. You could call it `checkFormal`.